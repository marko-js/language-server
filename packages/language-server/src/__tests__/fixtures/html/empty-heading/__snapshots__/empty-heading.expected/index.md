## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <h1></h1>
    |  ^^ Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  2 | <h2></h2>
  3 | <h3></h3>
  4 | <h4></h4>
```

### Ln 2, Col 2
```marko
  1 | <h1></h1>
> 2 | <h2></h2>
    |  ^^ Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  3 | <h3></h3>
  4 | <h4></h4>
  5 | <h5></h5>
```

### Ln 3, Col 2
```marko
  1 | <h1></h1>
  2 | <h2></h2>
> 3 | <h3></h3>
    |  ^^ Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  4 | <h4></h4>
  5 | <h5></h5>
  6 | <h6></h6>
```

### Ln 4, Col 2
```marko
  2 | <h2></h2>
  3 | <h3></h3>
> 4 | <h4></h4>
    |  ^^ Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  5 | <h5></h5>
  6 | <h6></h6>
```

### Ln 5, Col 2
```marko
  3 | <h3></h3>
  4 | <h4></h4>
> 5 | <h5></h5>
    |  ^^ Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  6 | <h6></h6>
```

### Ln 6, Col 2
```marko
  4 | <h4></h4>
  5 | <h5></h5>
> 6 | <h6></h6>
    |  ^^ Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
```

