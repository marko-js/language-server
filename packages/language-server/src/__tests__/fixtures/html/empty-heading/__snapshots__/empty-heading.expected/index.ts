export interface Input {}
abstract class Component extends Marko.Component<Input> {}
export { type Component };
(function (this: void) {
  const input = Marko._.any as Input;
  const component = Marko._.any as Component;
  const state = Marko._.state(component);
  const out = Marko._.out;
  const $signal = Marko._.any as AbortSignal;
  const $global = Marko._.getGlobal(
    // @ts-expect-error We expect the compiler to error because we are checking if the MarkoRun.Context is defined.
    (Marko._.error, Marko._.any as MarkoRun.Context),
  );
  Marko._.noop({ component, state, out, input, $global, $signal });
  Marko._.renderNativeTag("h1")()()({
    ["renderBody" /*h1*/]: (() => {
      return () => {
        return Marko._.voidReturn;
      };
    })(),
    /*h1*/
  });
  Marko._.renderNativeTag("h2")()()({
    ["renderBody" /*h2*/]: (() => {
      return () => {
        return Marko._.voidReturn;
      };
    })(),
    /*h2*/
  });
  Marko._.renderNativeTag("h3")()()({
    ["renderBody" /*h3*/]: (() => {
      return () => {
        return Marko._.voidReturn;
      };
    })(),
    /*h3*/
  });
  Marko._.renderNativeTag("h4")()()({
    ["renderBody" /*h4*/]: (() => {
      return () => {
        return Marko._.voidReturn;
      };
    })(),
    /*h4*/
  });
  Marko._.renderNativeTag("h5")()()({
    ["renderBody" /*h5*/]: (() => {
      return () => {
        return Marko._.voidReturn;
      };
    })(),
    /*h5*/
  });
  Marko._.renderNativeTag("h6")()()({
    ["renderBody" /*h6*/]: (() => {
      return () => {
        return Marko._.voidReturn;
      };
    })(),
    /*h6*/
  });
  return;
})();
export default new (class Template extends Marko._.Template<{
  render(
    input: Marko.TemplateInput<Input>,
    stream?: {
      write: (chunk: string) => void;
      end: (chunk?: string) => void;
    },
  ): Marko.Out<Component>;

  render(
    input: Marko.TemplateInput<Input>,
    cb?: (err: Error | null, result: Marko.RenderResult<Component>) => void,
  ): Marko.Out<Component>;

  renderSync(input: Marko.TemplateInput<Input>): Marko.RenderResult<Component>;

  renderToString(input: Marko.TemplateInput<Input>): string;

  stream(
    input: Marko.TemplateInput<Input>,
  ): ReadableStream<string> & NodeJS.ReadableStream;

  api: "class";
  _(): () => <__marko_internal_input extends unknown>(
    input: Marko.Directives &
      Input &
      Marko._.Relate<__marko_internal_input, Marko.Directives & Input>,
  ) => Marko._.ReturnWithScope<__marko_internal_input, void>;
}> {})();
