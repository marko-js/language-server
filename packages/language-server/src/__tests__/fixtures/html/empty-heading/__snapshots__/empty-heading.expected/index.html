<h1 data-marko-node-id="0"></h1><h2 data-marko-node-id="1"></h2><h3 data-marko-node-id="2"></h3><h4 data-marko-node-id="3"></h4><h5 data-marko-node-id="4"></h5><h6 data-marko-node-id="5"></h6>